src/services.js

```diff
@@ -0,0 +1,32 @@
+export const fetchCourses = () => {
+	return fetch('http://localhost:4000/courses/all')
```

Comment:
create a const for the "http://localhost:4000" and reuse this value in your API calls 
in the future, it will be much easier to change this URL in one place, than in 3 places you have (in the project there could be more than 10 endpoints)

const BASE_API_URL = "http://localhost:4000";

```suggestion
	return fetch(`${BASE_API_URL}/courses/all`)
```